cfpb/cfgov-refresh
# Fix Spanish translation of "required" in form-field-with-button

## Changes

- Corrects the Spanish translation of "required" for the Form Field with Button.

## Testing

1. Pull branch
2. Runserver
3. Create an Email Sign-up block, mark the field as required, and set that page's language to Spanish
4. Preview the page and see the updated text

## Checklist
